### DISCUSSION TITLE
app::background design

### DISCUSSION THREAD

**Discussion by wyhinton** [fltk-rs/fltk-rs#787] [Q&A]
Just wondering, why does ```app::background()``` take an (r,g,b) value instead of a ```Color```? 

```app::background(50, 50, 50)```

**Comment by MoAlyousef** [comment]
Fl::background doesn't take a Fl_Color, and if you convert an RGB value to an Fl_Color (RGBI), as fltk-rs does, the last bits are misread if you pass it Fl::background. It’s the same issue with trying to acquire an RGB value from an Fl_Color, which for the enum colors, only the index bits have a value and the RGB bits are zeroes.
